From 843204c3b705ed4dff65e79b674d0312b843be94 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@willowgarage.com>
Date: Thu, 18 Apr 2013 12:06:21 +0300
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 76fc83f..f54a696 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -159,3 +159,5 @@ if (OMPL_REGISTRATION)
         endif()
     endif()
 endif()
+
+install(FILES package.xml DESTINATION "share/ompl${OMPL_INSTALL_SUFFIX}")
-- 
1.7.9.5

