From c8093329ba9ce0f1aa9a415f6446a31f84374be9 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@willowgarage.com>
Date: Thu, 18 Apr 2013 11:56:46 +0300
Subject: [PATCH] Added install rule for package.xml

---
 CMakeLists.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f6bcb00..7e67786 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -142,3 +142,5 @@ if (OMPL_REGISTRATION)
         endif()
     endif()
 endif()
+
+install(FILES package.xml DESTINATION "share/ompl${OMPL_INSTALL_SUFFIX}")
-- 
1.7.9.5

