From dd2891774cb982215d0d0cf1d52475860f30aef7 Mon Sep 17 00:00:00 2001
From: Mark Moll <mark@picknik.ai>
Date: Wed, 27 Nov 2019 10:31:39 -0600
Subject: [PATCH] fix check for package.xml

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fbde81b..bb91e43 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -243,7 +243,7 @@ if (OMPL_REGISTRATION)
 endif()
 
 # needed for ROS installation
-if (EXISTS "package.xml")
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/package.xml")
     # install catkin package.xml
     install(FILES package.xml DESTINATION "share/ompl")
 endif()
-- 
2.17.1

