From b9d72239219911b227d54b448f0dacdc795dfc8f Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 20:10:56 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b4c17a9..5312fcb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -203,3 +203,5 @@ if (OMPL_REGISTRATION)
         endif()
     endif()
 endif()
+
+install(FILES package.xml DESTINATION "share/ompl${OMPL_INSTALL_SUFFIX}")
-- 
2.1.4

