From 210ec133323df2a3971409fe14e13379f196e738 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 20:10:56 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 83015a1..bacf2c8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -206,3 +206,5 @@ if (OMPL_REGISTRATION)
         endif()
     endif()
 endif()
+
+install(FILES package.xml DESTINATION "share/ompl${OMPL_INSTALL_SUFFIX}")
-- 
1.9.1.423.g4596e3a

