From aafd58e8b9e6ea32158a46f8be83c1cd7d70e128 Mon Sep 17 00:00:00 2001
From: Richard Bain <richard.j.bain2@boeing.com>
Date: Thu, 18 Jan 2018 15:38:57 +1100
Subject: [PATCH] Added package.xml install rule in CMakeLists.txt

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c7d92ca..27d1f88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -212,3 +212,6 @@ if (OMPL_REGISTRATION)
         endif()
     endif()
 endif()
+
+# install catkin package.xml
+install(FILES package.xml DESTINATION "share/ompl${OMPL_INSTALL_SUFFIX}")
-- 
2.7.4

