From a78d79f295e9c1da1b9aef6fdd911c92d08f1fba Mon Sep 17 00:00:00 2001
From: Mark Moll <mark@picknik.ai>
Date: Wed, 11 Dec 2019 21:50:41 -0600
Subject: [PATCH 3/3] fix cmake error

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3b5fca6..071da5f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -243,7 +243,7 @@ if (OMPL_REGISTRATION)
 endif()
 
 # needed for ROS installation
-if (EXISTS "${CURRENT_SOURCE_DIR}/package.xml")
+if (EXISTS "${CMAKE_SOURCE_DIR}/package.xml")
     # install catkin package.xml
     install(FILES package.xml DESTINATION "share/ompl")
 endif()
-- 
2.17.1

